src/domains/login/page/KakaoSuccess.jsx

```
@@ -1,6 +1,6 @@
-import { useEffect } from 'react';
-import { useLocation, useNavigate } from 'react-router-dom';
-import ToastService from '../../../utils/toastService';
+import { useEffect } from 'react'
```

ESLint

[error] 'useEffect' is defined but never used.

(no-unused-vars)